website/content/docs/connect/native/go.mdx

```diff
@@ -7,6 +7,9 @@ description: >-
 
 # Service Mesh Native Integration for Go Applications
 
+-> **Note:** The Connect Native golang SDK is currently deprecated and will be removed in a future Consul release.
+The eventual removal of the SDK is dependent on the delivery of proxyless GRPC via XDS which is tracked on [GH-10339](https://github.com/hashicorp/consul/issues/10339).
```

Comment:
```suggestion
We will replace the SDK functionality with proxy-less gRPC connections to Consul xDS. Refer to [GH-10339](https://github.com/hashicorp/consul/issues/10339) for additional information and to track progress toward the replacement functionality.
```
Is it correct to characterize the upcoming functionality as a replacement for the SDK?

Reply:
Technically yes since all that is needed is the native golang package for XDS. I.e.
```
import _ "google.golang.org/grpc/xds"
```